cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -596,6 +598,9 @@ func (csr *ClusterStateRegistry) updateReadinessStats(currentTime time.Time) {
 		}
 		perNodeGroup[nodeGroup.Id()] = perNgCopy
 	}
+	if total.LongUnregistered > 0 {
+		klog.Warningf("Found longUnregistered Nodes %s", longUnregisteredNodeNames)
```

Comment:
The list can get quite long on larger clusers (in case of some outage) and printing it every iteration may be problematic. Please use higher lever of info for printing this.  Frequenlyy printend warnings/errors should contain only relatively brief messages (like the number of nodes, not their list).

Reply:
Sure, I can log it at Info level instead.

In #4001 discussion, it was explained that this information is very difficult to obtain from any other source. Asking cluster operators to query this information from the cloud API and compare to the Kubernetes API is cumbersome, and the data already exists in `clusterstate.go`. Adding this node-level data to metrics has a performance impact on metrics. A long log line is the least impactful way to make this information available to cluster operators. In my organization we get a lot of feedback from our ops teams on the difficulty of troubleshooting unregistered nodes without this information. I think the tradeoff of a long log line in the uncommon case of an outage is worth the human time and effort saved in the common case where a relatively small number of nodes are unregistered.